.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,10 @@ Please read the [Code of Conduct](https://github.com/nodejs/nodejs.org/blob/main
 
 <!-- Write a brief description of the changes introduced by this PR -->
 
+## Validation
```

Comment:
IMO the term `Validation` is a bit too broad, it can mean any Validation.
I would say something like "Test Plan" suits better in this case.
The benefit of having a Test Plan is that we encourage people to test the changes first before Code Review.

However, I am wondering if it's a necessary step. Why? Because a Preview will be auto deployed (we can see the Preview instead). And after seeing the PR you're referencing, I think that the description can be made clearer hence there's no need to have a Test Plan in the first place.

Reply:
Adding more...

But if we have something that can't be covered by Preview, e.g. result from running a command. Then, having a Test Plan as a necessary step will benefit us.

Reply:
happy to rename it. the goal is to make things WAY more clear what was broke, how you fixed it, and how reviewers will know its fixed. The automated deployments arent enough. See https://github.com/nodejs/nodejs.org/pull/5500#pullrequestreview-1525353617

The burden of proof that a change is valid is on the pull request author. We all have too much to do as reviewers to guess.

Reply:
> The burden of proof that a change is valid is on the pull request author.

Agree on this.

It helps the reviewers to recall the necessary contexts as well.